tracer/src/Datadog.Trace/Configuration/ExporterSettings.cs

```diff
@@ -196,6 +248,35 @@ private void ConfigureTraceTransport(IConfigurationSource source, bool isWindows
                     ?? 500;
 #endif
             }
+            else
+            {
+                // Check for UDS
+                if (isConventionBasedSocket)
+                {
+                    traceTransport = TracesTransportType.UnixDomainSocket;
+                    TracesUnixDomainSocketPath = agentHost;
+                }
+                else
+                {
+                    // check for explicit UDS config
```

Comment:
I feel this should be the first check.

Reply:
The code here was modified and I will be adding another test to the exporter settings.
The convention based socket counts as explicit config IMO and the RFC does not actually differentiate between explicit config priority.